ci/packaging/suse/skuba_spec_template

```diff
@@ -71,13 +72,14 @@ including patches requiring reboot.
 %prep
 %setup -q -n %{name}
 %patch0 -p1
+%setup -D -a 1 -n %{name}
 
 %build
 # TAG and CLOSEST_TAG envvars are populated so the Makefile will read this information
 export TAG="%%TAG"
 export CLOSEST_TAG="%%CLOSEST_TAG"
 export GOMODFLAG=-mod=vendor
-mkdir -pv $HOME/go/src/%{go_project}
+mkdir -pv $HOME/go/src/%{go_project}/vendor
```

Comment:
This isn't needed, it's done by the `%setup` line you added

Reply:
changed